src/scores/categorical/contingency_impl.py

```diff
@@ -1131,10 +1213,15 @@ def make_contingency_manager(
 
 class ThresholdEventOperator(EventOperator):
     """
-    Given a forecast and and an observation, consider an event defined by
-    particular variables meeting a threshold condition (e.g. rainfall above 1mm).
+    See https://scores.readthedocs.io/en/stable/tutorials/Binary_Contingency_Scores.html for
+    a detailed walkthrough showing the use in practice.
+
+    A ThresholdEventOperator is used to produce a :py:class:`BinaryContingencyManager` from
+    forecast and observed data. It considers an event to be defined when the forecast and
+    observed variables meet a particular threshold condition (e.g. rainfall above 1mm).
```

Comment:
Is 1mm a suitable measurement in this context - would 1mm typically be a threshold of interest for rainfall? (I genuinely don't know the answer, I am just asking).

Reply:
I think using 1mm is sensible. It's a threshold that some users of the forecasts definitely care about